Products/RhaptosModuleEditor/skins/rhaptos_module_editor/publishContent.cpy

```diff
@@ -39,6 +39,37 @@ if hasattr(context,'excludedIds'):
     if excludedexists:
         context.manage_delObjects(excludedexists)
 
+# Generate SVGs for MathML
+import requests
```

Comment:
Do `requests` and `lxml` need to be "allowed" in order to be used in a skin script?